### DISCUSSION TITLE
about hover to child elements

### DISCUSSION THREAD

**Discussion by leosin** [unocss/unocss#3653] [General]
We often encounter scenarios in element interactions where, as the mouse hovers over a .demo element, the h3 and p elements change to different colors. Typically, this is achieved by adding different styles to the child elements under .demo:hover. So, is there a way to achieve this directly within the template using UnoCSS's syntax?

```vue
<div class="demo">
  <h3>title</h3>
  <p>content</p>
</div>
<style lang="scss" scoped>
.demo:hover{
  h3{ color:red;  }
  p{ color:blue;  }
}
</style>
```

**Comment by leosin** [comment]
Does anyone know the answer?

**Reply by schelmo** [reply]
```html
<div class="group">
  <h3 class="group-hover:bg-red">title</h3>
  <p class="group-hover:bg-blue">content</p>
</div>
```

**Reply by leosin** [reply]
thank you very much
